fulp_modules/bloodsuckers/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -0,0 +1,479 @@
+//									IDEAS		--
+//					An object that disguises your coffin while you're in it!
+//
+//					An object that lets your lair itself protect you from sunlight, like a coffin would (no healing tho)
+
+
+
+// Hide a random object somewhere on the station:
+//		var/turf/targetturf = get_random_station_turf()
+//		var/turf/targetturf = get_safe_random_station_turf()
+
+
+
+
+// 		CRYPT OBJECTS
+//
+//
+// 	PODIUM		Stores your Relics
+//
+// 	ALTAR		Transmute items into sacred items.
+//
+//	PORTRAIT	Gaze into your past to: restore mood boost?
+//
+//	BOOKSHELF	Discover secrets about crew and locations. Learn languages. Learn marial arts.
+//
+//	BRAZER		Burn rare ingredients to gleen insights.
+//
+//	RUG			Ornate, and creaks when stepped upon by any humanoid other than yourself and your vassals.
+//
+//	X COFFIN		(Handled elsewhere)
+//
+//	X CANDELABRA	(Handled elsewhere)
+//
+//	THRONE		Your mental powers work at any range on anyone inside your crypt.
+//
+//	MIRROR		Find any person
+//
+//	BUST/STATUE	Create terror, but looks just like you (maybe just in Examine?)
```

Comment:
well, we have the new chiseling code now- you could just make it so that if a vampire chisels a statue of themself (or a vassal chisels a statue of their master), the statue gains a terror-inducing property or something